examples/cassandra/README.md

```diff
@@ -53,54 +53,48 @@ This is a somewhat long tutorial.  If you want to jump straight to the "do it no
 In Kubernetes, the atomic unit of an application is a [_Pod_](../../docs/user-guide/pods.md).  A Pod is one or more containers that _must_ be scheduled onto the same host.  All containers in a pod share a network namespace, and may optionally share mounted volumes.
 In this simple case, we define a single container running Cassandra for our pod:
 
-<!-- BEGIN MUNGE: EXAMPLE cassandra-controller.yaml -->
+<!-- BEGIN MUNGE: EXAMPLE cassandra.yaml -->
 
 ```yaml
 apiVersion: v1
-kind: ReplicationController
+kind: Pod
```

Comment:
@jtblin Did you read the document I posted a link to? Please do, and also:
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/pods.md#durability-of-pods-or-lack-thereof

I would prefer not to show an example that lead users astray. If we show a pod as the first example, we need to make it clear that a pod is insufficient to ensure self-healing, as part of the motivation for using a ReplicationController in the following section. Nobody should run with a naked pod in production. Even `kubectl run` doesn't create naked pods.